coral/src/app/pages/index.tsx

```diff
@@ -1,15 +1,30 @@
 import { useQuery } from "@tanstack/react-query";
+import { AuthUser, getAuthUser } from "src/domain/auth-user";
+import { mockUserAuthRequest } from "src/domain/auth-user/auth-user-api.msw";
+import { useEffect } from "react";
+import { getWindowWithMswInstance } from "src/services/api-mocks/window-msw";
 
 const HomePage = () => {
-  const { isLoading, data } = useQuery(["testData"], () => {
-    return Promise.resolve({ data: "hello" });
-  });
+  useEffect(() => {
+    const window = getWindowWithMswInstance();
+    const browserEnvWorker = window.msw;
+
+    if (browserEnvWorker) {
+      const mswInstance = browserEnvWorker;
```

Comment:
nit: here we are assigning the same value into multiple variables -> could be streamlined a bit.

Reply:
Makes sense!